css/style.css

```diff
@@ -0,0 +1,322 @@
+.container {
+  font-family: sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.header {
+  width: 360px;
+  background-image: url("../images/background.png");
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  width: 360px;
+  padding-left: 12px;
+}
+
+.navbar span {
+  margin: auto 0;
+}
+
+.navbar > * {
+  padding: 20px;
+}
+
+.navbar a {
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  color: #28352f;
+  text-decoration: none;
+}
+
+.header-title {
+  text-align: center;
+}
+
+.header-title h1,
```

Comment:
- [ ] please use the correct font family of 'Inter'

Reply:
> ok
[OPTIONAL] Because I didn't mention this previously,
you also need to import 'Inter' from Google fonts and put it at the top of your css file because sans-serif isn't natively on the user's machine

## EXAMPLE
`@import url('https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600&display=swap');`